docs/en/8-MicroSim/GettingStarted.md

```diff
@@ -0,0 +1,125 @@
+
+# Getting started with Kubeflow Notebooks and OpenM++.
+
+This document is intended to act as an introduction to The AAWs Kubeflow Notebook servers and their use for the OpenM++ project.
+
+
+## Starting
+
+To access the AAW Kubeflow portal, navigate to the following website.
+
+[https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca/) 
+
+This will redirect you to a Microsoft log-in page.
+
+![Login01](../images/Login01.png)
+
+Select the account you wish to use and proceed with the authentication.
+
+After your credentials are authenticated, you will be redirected to the AAW Kubeflow management panel. 
+
+![Kube Flow Management Panel](../images/KFMP01.png)
+
+## Create a notebook.
+
+Click on the Create a New Notebook server button.
+
+![Create Notebook01](../images/CreateNB01.png)
+
+This brings up the new Notebook screen.
+
+To create a new Notebook, three things must be set:
+- Ensure the correct Namespace is selected.
+- A unique name in the name field. A unique name is automatically generated in the Name field when you click in it,
+- then click on the Notebook type you want.
+  - For OpenM++, select the JupyterLab option.
+- If you are working with Protected B Data, select the **Run in Protected B notebook** checkbox
+
+Scroll down to see the **Launch Button.**  The Launch button will only be active after the above options are selected.
+ 
+![Notebook screen](../images/NewNBScreen02.png)
+
+Press the Launch button to launch you new Notebook.
+
+[For more information about Statcan AAW Kubeflow, Click here.](https://statcan.github.io/aaw/en/1-Experiments/Kubeflow/)
+
+### Existing Notebooks
+
+If you have previously created a notebook, you can reuse it by clicking on the Notebooks Tab.
+
+![Notebook screen](../images/NewNBScreen03.png)
+
+This will bring up a window with all your existing Notebooks.  
+
+![Notebook screen](../images/NewNBScreen04.png)
+
+To start an existing Notebook, select it and press the CONNECT button.
+
+If the **Connect button** is disabled, click on the triangle (Start image) button to start the image, and then click on **connect** when it becomes avaliable .
```

Comment:
Should we also advise the user that they should Stop their notebook when they have finished using it? Otherwise they will continue to be charged for the idle resource usage.